proxy/agent.go

```diff
@@ -78,6 +78,12 @@ var (
 	}
 )
 
+const (
+	// MaxAborts is the maximum number of cascading abort messages to buffer.
+	// This should be the upper bound on the number of proxies available at any point in time.
+	MaxAborts = 10
```

Comment:
there are up to "n" proxies around. if (n-1) crash together, proxy #0 gets n-1 aborts. 10 is a safe upper bound. there're at most 3 envoy processes active at a time.